assets/linux-deb/conffiles

```diff
@@ -0,0 +1,2 @@
+/etc/sunshine/sunshine.conf
```

Comment:
`sunshine.conf` probably doesn't even need to be included (or even exist in the repo) anymore, as I cleared it out and moved all the documentation to the sphinx docs. What do you think?

Reply:
It's your call, even if it's safe to be removed it could be helpful for people to look at the defaults..

Let me know what you prefer!

Reply:
These are the contents of the `sunshine.conf` after I moved everything to the docs.

```txt
# See our documentation at https://sunshinestream.readthedocs.io/en/latest/about/advanced_usage.html
```

Reply:
Based on our discord conversation, let's leave the `sunshine.conf`.